### PR TITLE
Fix grammatical and wording errors in - README.md

### PR DESCRIPTION
**Pull request by gavinmiller**
A few grammatical and wording changes, hope this helps :)

- Gavin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
-Every day, millions of people come up with ideas that have the potential to change the world. Yet almost all ideas do not take any form mainly because of lack of support from peers with similar vision or the originator does not have a clear path of how to make it a reality. 
+Every day millions of people come up with ideas that have the potential to change the world. Yet almost all ideas do not take any form, mainly because of lack of support from peers with similar vision, or the originator does not have a clear path of how to make it a reality. 
 
 > ## We exist to take your ideas to execution. 
 
-We are different from any freelancing program as in the process, you learn new skills and thousands of passionate people are beside your shoulder for the entire journey. You do not owe a penny to us. 
+We are different from any freelancing program as in the process you learn new skills and thousands of passionate people are beside your shoulder for the entire journey. You do not owe a penny to us. 
 
-> We do everything to bring a smile on your face. 😄 
+> We do everything to bring a smile to your face. 😄
```
Comment (arnavb) [inline README.md]: This line was fine as it is.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 
 This is how it works: 
 
 ![alt text](https://github.com/AdiChat/Freelance/blob/master/data/Promo.jpg "OpenGenus Freelance")
 
 * Join us at OpenGenus by opening an issue [here](https://github.com/OpenGenus/Join_OpenGenus)
-* You will be welcomed to our community in a day. 
-* Once you join, create an issue or get in touch at **Slack**. Tell us your vision and what you want to do. 
-* We will direct you to **free** resources on the web to attain essential skills to make your vision a reality. We will, also, guide you through the entire course. We will try to keep this period short as much as a month. 
+* You will be welcomed intto our community within a day.
```
Comment (arnavb) [inline README.md]: Typo.

File: README.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
 This is how it works: 
 
 ![alt text](https://github.com/AdiChat/Freelance/blob/master/data/Promo.jpg "OpenGenus Freelance")
 
 * Join us at OpenGenus by opening an issue [here](https://github.com/OpenGenus/Join_OpenGenus)
-* You will be welcomed to our community in a day. 
-* Once you join, create an issue or get in touch at **Slack**. Tell us your vision and what you want to do. 
-* We will direct you to **free** resources on the web to attain essential skills to make your vision a reality. We will, also, guide you through the entire course. We will try to keep this period short as much as a month. 
+* You will be welcomed intto our community within a day. 
+* Once you join, create an issue or get in touch on **Slack**. Tell us your vision and what you want to do.
```
Comment (arnavb) [inline README.md]: _at_ was correct. However, I would change to: ... get in touch _with us at_ Slack.

File: README.md [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
 
 ![alt text](https://github.com/AdiChat/Freelance/blob/master/data/Promo.jpg "OpenGenus Freelance")
 
 * Join us at OpenGenus by opening an issue [here](https://github.com/OpenGenus/Join_OpenGenus)
-* You will be welcomed to our community in a day. 
-* Once you join, create an issue or get in touch at **Slack**. Tell us your vision and what you want to do. 
-* We will direct you to **free** resources on the web to attain essential skills to make your vision a reality. We will, also, guide you through the entire course. We will try to keep this period short as much as a month. 
+* You will be welcomed intto our community within a day. 
+* Once you join, create an issue or get in touch on **Slack**. Tell us your vision and what you want to do. 
+* We will direct you to **free** resources on the web to attain essential skills to make your vision a reality. We will also guide you through the entire course and will try to keep this period as short as a month.
```
Comment (arnavb) [inline README.md]: The `will` after the `and` can be removed. period _to_ as short as a month.

File: README.md [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
+* Once you join, create an issue or get in touch on **Slack**. Tell us your vision and what you want to do. 
+* We will direct you to **free** resources on the web to attain essential skills to make your vision a reality. We will also guide you through the entire course and will try to keep this period as short as a month. 
  
  > Yes, we will arrange your access to great resources.
 
-* Meanwhile, OpenGenus members, even from the remotest part of the World, take a look at your vision. People with similar passion and concern reach out to you and join in the mission.
-* Once, you are ready, we build a prototype of your vision and provide all resources as might be required. 
-* Within a small period, we will see the impact of your work due to the large scale of our project. 
-* Our love and journey continue forever. 
+* Meanwhile, OpenGenus members, even from the most remote part of the World, will take a look at your vision. People with similar passion and concern reach out to you and join in the mission.
```
Comment (arnavb) [inline README.md]: Remove the comma before `will`.

File: README.md [hunk 6]
```diff
@@ -1,25 +1,25 @@
...
+* Once you are ready, we will build a prototype of your vision and provide all necessary resources that might be required.
```
Comment (arnavb) [inline README.md]: How about: Once we think you're ready, ...

File: README.md [hunk 7]
```diff
@@ -1,25 +1,25 @@
...
  
  > Yes, we will arrange your access to great resources.
 
-* Meanwhile, OpenGenus members, even from the remotest part of the World, take a look at your vision. People with similar passion and concern reach out to you and join in the mission.
-* Once, you are ready, we build a prototype of your vision and provide all resources as might be required. 
-* Within a small period, we will see the impact of your work due to the large scale of our project. 
-* Our love and journey continue forever. 
+* Meanwhile, OpenGenus members, even from the most remote part of the World, will take a look at your vision. People with similar passion and concern reach out to you and join in the mission.
+* Once you are ready, we will build a prototype of your vision and provide all necessary resources that might be required. 
+* Within a small time period, we will see the impact of your work due to the large scale of our project.
```
Comment (arnavb) [inline README.md]: I think this sentence is better phrased as:

> Within a small period of time, we will see the impact of _your hard_ work due to the large scale of this project.

File: README.md (hunk 1 above)
Comment (gavinmiller) [reply README.md]: I'm not saying it's a big deal, it's just that the expression is 'bring a smile to [my/your] face'.

File: README.md (hunk 2 above)
Comment (gavinmiller) [reply README.md]: Cheers!

File: README.md (hunk 1 above)
Comment (arnavb) [reply README.md]: @gavinmiller If you say so. The old version just sounded better to me.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

positive_opengenus_first

**Pull request merged**